Cargo.toml

```diff
@@ -36,8 +36,8 @@ version = "0.16.0-soroban2"
 git = "https://github.com/stellar/wasmi"
 rev = "862b32f5"
 
-# [patch."https://github.com/stellar/rs-stellar-xdr"]
-# stellar-xdr = { path = "../rs-stellar-xdr/" }
+[patch."https://github.com/stellar/rs-stellar-xdr"]
+stellar-xdr = { path = "../rs-stellar-xdr/" }
```

Comment:
Just a reminder to revert this before merging.

Reply:
done